### PR TITLE
feat(*): add clause to contract

### PR DESCRIPTION
**Pull request by DianaLease**
https://github.com/accordproject/template-studio-v2/issues/25
feat: implement add clause to contract button
feat: persist changes to Redux store and keep in sync with Slate editor

### CODE REVIEW

File: src/actions/contractActions.js
Comment (jolanglinais) [inline src/actions/contractActions.js:13]: Should we handle this in the reducer? That would already be set up for the `Error` component to handle it in the future.

**Review by jolanglinais — COMMENTED** [review]
I tried to re-run the Netlify checks. This has happened to me a couple times recently where it times out and then works later, so I'm unsure if it's the code or just Netlify.

File: src/actions/contractActions.js
Comment (DianaLease) [reply src/actions/contractActions.js:13]: Yes, meant to add that. We should decide how we want to handle these types of errors that are not cicero/ergo errors in the UI.

Comment (jolanglinais) [reply src/actions/contractActions.js:13]: Right. I think this shouldn't hold this PR, and can be a separate issue/feature in the future.

**Review by jolanglinais — APPROVED** [review]
Great!

**Pull request merged**
